closure/protobuf/BUILD

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 The Closure Rules Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+load("//closure/compiler:closure_js_library.bzl", "closure_js_library")
+
+closure_js_library(
+    name = "protobuf",
```

Comment:
Also it might be worth moving //closure/protobuf to //protobuf which is probably the directory we'll want to use when we make rules_protobuf